lib/dialects/sqlite/query-interface.js

```diff
@@ -106,3 +107,63 @@ function renameColumn(tableName, attrNameBefore, attrNameAfter, options) {
   });
 }
 exports.renameColumn = renameColumn;
+
+function removeConstraint(tableName, constraintName, options) {
+  let createTableSql;
+  /* jshint validthis:true */
+  return this.showConstraint(tableName, constraintName)
+    .then(constraints => {
+      const constraint = constraints[0];
+
+      if (constraint) {
+        createTableSql = constraint.sql;
+        constraint.constraintName = this.QueryGenerator.quoteIdentifier(constraint.constraintName);
+        let constraintSnippet = `, CONSTRAINT ${constraint.constraintName} ${constraint.constraintType} ${constraint.constraintCondition}`;
+        
+        if (constraint.constraintType === 'FOREIGN KEY') {
+          const referenceTableName = this.QueryGenerator.quoteTable(constraint.referenceTableName);
+          constraint.referenceTableKeys = constraint.referenceTableKeys.map(columnName => this.QueryGenerator.quoteIdentifier(columnName));
+          const referenceTableKeys = constraint.referenceTableKeys.join(', ');
+          constraintSnippet += ` REFERENCES ${referenceTableName} (${referenceTableKeys})`;
+          constraintSnippet += ` ON UPDATE ${constraint.updateAction}`;
+          constraintSnippet += ` ON DELETE ${constraint.deleteAction}`;
+        }
+
+        createTableSql = createTableSql.replace(constraintSnippet, '');      
+        createTableSql += ';';
+        return this.describeTable(tableName, options);
```

Comment:
New line above `return`, Please fix this for other sections that you have changed as well